(dp1
S'output'
p2
S"<type 'exceptions.AttributeError'> 'DAL' object has no attribute 'escultor'"
p3
sS'layer'
p4
S'/home/informatorio/Documentos/Facus/web2py/applications/EsQltoR/controllers/EscultorController.py'
p5
sS'code'
p6
S"# -*- coding: utf-8 -*-\n# Controlador de la Lista de Escultores.\ndef index():\n    escultores = db().select(db.t_escultores.ALL,orderby=db.t_escultores.id)\n    return dict(escultores=escultores)\n\ndef show():\n    escultor = db.escultor(request.args(0)) or redirect(URL('index'))\n    return dict(escultor=escultor)\n\nresponse._vars=response._caller(show)\n"
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/informatorio/Documentos/Facus/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/informatorio/Documentos/Facus/web2py/applications/EsQltoR/controllers/EscultorController.py", line 11, in <module>\n  File "/home/informatorio/Documentos/Facus/web2py/gluon/globals.py", line 417, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/informatorio/Documentos/Facus/web2py/applications/EsQltoR/controllers/EscultorController.py", line 8, in show\n    escultor = db.escultor(request.args(0)) or redirect(URL(\'index\'))\n  File "/home/informatorio/Documentos/Facus/web2py/gluon/packages/dal/pydal/base.py", line 921, in __getattr__\n    return BasicStorage.__getattribute__(self, key)\nAttributeError: \'DAL\' object has no attribute \'escultor\'\n'
p11
s.